modules/stochastic_tools/test/tests/reporters/BFActiveLearning/gold/multiple_proc_multiple_row_ufunction.json

```diff
@@ -0,0 +1,1102 @@
+{
```

Comment:
For future tests. Keep the `json` output at a reasonable level. It can get really verbose really fast. You might not need to run 20 time steps to verify that your model runs and gives the expected results.

Reply:
I have reduced the number of samples to keep the json smaller.